data/tim.uper

```diff
@@ -0,0 +1 @@
+001F80A97014B901EC9C236B00000000000F775D9B0301EA73E452D1539716C99E9D555100003F0A59B080010007F8AA9979F4D3BB3A0A9266C000000854E3B2C47291F21E85EEF057980028422C1FFE0001FFFC00017FFF80000FFFFF000009FFFFF8000005FFFFFF00000007FF80007FFF00005FFFE00003FFFFC000027FFFFE0000017FFFFFC000000C523E43D138ECB11C0000C00FB0473DFB72A0E997C74007D408E5C376CD4F775D9B00
```

Comment:
Let's remove and add a .gitignore entry for the uper so we can test it

Reply:
Removed this file & added a .gitignore entry for uper files